examples/CMakeLists.txt

```diff
@@ -12,8 +12,9 @@ macro(add_example EXAMPLE_NAME EXAMPLE_SOURCE)
   add_executable(${EXAMPLE_NAME} ${STATIC_OBJECTS} ${EXAMPLE_SOURCE} ${ARGN})
   target_link_libraries(${EXAMPLE_NAME} open62541 ${open62541_LIBRARIES})
   assign_source_group(${EXAMPLE_SOURCE})
-  add_dependencies(${EXAMPLE_NAME} open62541_amalgamation_header open62541_amalgamation_source)
+  add_dependencies(${EXAMPLE_NAME} open62541_amalgamation)
```

Comment:
Ok. Thanks.
Now I'm waiting for @Infinity95 to make a final merge of master into #1003 .
Then, the merge will happen in 2 or 3 waves.